src/clj_kondo/impl/analyzer.clj

```diff
@@ -1817,6 +1816,8 @@
                                     :end-row (:end-row expr-meta)
                                     :col col
                                     :end-col (:end-col expr-meta)
+                                    :scope-end-row (:end-row expr-meta)
```

Comment:
@ericdallo Why are you adding twice the same data, once as `:end-row` and once as `:scope-end-row` (same for col)?

Reply:
because only this place we want to add the scope, If I use the same end-row [here](https://github.com/clj-kondo/clj-kondo/pull/1387/files#diff-19afea30102cec0da2c4d0516d5e834219f6ccaadc2019a9b86e5af604050425R232) other usages will have scope and [this test](https://github.com/clj-kondo/clj-kondo/pull/1387/files#diff-f73159ab0c7e8a8b4d7d99a2b007cdc86046bb5dd44951e0f129bc43c1505a66R370-R383) will fail